src/components/policy/policy_regular/test/sql_pt_representation_test.cc

```diff
@@ -1106,17 +1187,22 @@ TEST_F(SQLPTRepresentationTest,
 TEST_F(SQLPTRepresentationTest,
        GetInitialAppData_SetData_ExpectCorrectValuesReceived) {
   // Arrange
-  const char* query_insert =
+  const std::string query_insert1 =
```

Comment:
@ZhdanovP please give more descriptive names like `insert_nickname_query`, `insert_app_type_query` and so on

Reply:
@AKalinich-Luxoft The commits were updated.